packages/superdough/superdough.mjs

```diff
@@ -190,11 +190,18 @@ export function getAudioContextCurrentTime() {
   return getAudioContext().currentTime;
 }
 
+let externalWorklets = [];
+export function registerWorklet(url) {
```

Comment:
i've added this to be able to register worklets from the outside without depending on them from superdough